app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Good job working on this method, but it will be nicer if you implemented a Rails Active Callback `after_save` to call the `update_like_counter` method every time a new post has been saved to the database, then make the `update_like_counter` method private. This will update the count every time a new like is saved.

Reply:
Thank you for the feedback. Done